data/remnant/remnant 1 introduction.txt

```diff
@@ -126,40 +126,28 @@ mission "First Contact: Remnant"
 
 
 
-# Compatibility patch for those who made contact with the Remnant prior to v0.9.9
-# This will also catch everyone who completed this mission prior to the correction in 0.9.15 to give them "remnant: blood test pure"
+# Compatibility patch for those who made contact with the Remnant prior to v0.9.15:
 mission "Remnant: Blood Test Patch"
 	landing
 	invisible
 	to offer
 		has "event: ember waste label"
 	on offer
-		set "remnant blood test pure" # This line is depreciated and will be removed in future versions. Test for "remnant: blood test pure" instead.
 		set "remnant: blood test pure"
-		fail
-
-
-
-# Changes Remnant hails once contacted and submitted to blood test
-mission "Remnant: Hails Change"
-	landing
-	invisible
-	to offer
-		has "remnant: blood test pure"
-	on offer
 		event "remnant: change hail"
 		fail
 
 
 
-mission "Remnant: Lost in Ember Waste 1"
+mission "Remnant: Lost in Ember Waste: Trigger"
 	landing
 	invisible
 	source
 		government "Remnant"
 	to offer
 		has "remnant: blood test pure"
 		not "license: Remnant"
+		not "lost in ember waste: offered"
```

Comment:
We technically don't need this either. If the old mission already triggered then you also already got the mission that gives you the keystone, but that mission name hasn't changed so even if the trigger mission is updated it can still only offer once.
```suggestion
		not "lost in ember waste: offered"
```